extensions/wikia/PhalanxII/fallback/hooks/UserBlock.class.php

```diff
@@ -225,14 +225,14 @@ public static function getBlockReasonMessage( $reason, $isExact, $isBlockIP ) {
 		return $result;
 	}
 
-	//moved from RegexBlockCore.php
-	private static function setUserData(User $user, $blockData, $address /* not used at all */, $isBlockIP = false) {
+	// moved from RegexBlockCore.php
+	private static function setUserData( User $user, $blockData, $address /* not used at all */, $isBlockIP = false ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$address". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)